Utilities/bootstrap

```diff
@@ -261,6 +283,12 @@ class Target(object):
             import_paths.append(os.path.join(args.libdispatch_build_dir, "src", "swift"))
         if args.libdispatch_source_dir:
             import_paths.append(args.libdispatch_source_dir)
+            if platform.system() == 'Darwin':
+                other_args.extend(["-Xcc", "-DDEPLOYMENT_TARGET_MACOSX"])
+            elif platform.system() == 'Linux':
+                other_args.extend(["-Xcc", "-DDEPLOYMENT_TARGET_LINUX"])
+            elif platform.system() == 'Windows':
```

Comment:
You added DEPLOYMENT_TARGET_FREEBSD down below, so add it here as well. Also, I'd move this into a function with returns the appropriate DEPLOYMENT_TARGET_* macro name given a `platform.system()` output string.